website/settings/defaults.py

```diff
@@ -328,1587 +154,15 @@ def parent_dir(path):
 GOOGLE_ANALYTICS_ID = None
 GOOGLE_SITE_VERIFICATION = None
 
-DEFAULT_HMAC_SECRET = 'changeme'
-DEFAULT_HMAC_ALGORITHM = hashlib.sha256
 WATERBUTLER_URL = 'http://localhost:7777'
 WATERBUTLER_INTERNAL_URL = WATERBUTLER_URL
 WATERBUTLER_ADDRS = ['127.0.0.1']
 
-# Test identifier namespaces
-DOI_NAMESPACE = 'doi:10.5072/FK2'
-ARK_NAMESPACE = 'ark:99999/fk4'
-
-# For creating DOIs and ARKs through the EZID service
-EZID_USERNAME = None
-EZID_PASSWORD = None
-# Format for DOIs and ARKs
-EZID_FORMAT = '{namespace}osf.io/{guid}'
-
-# Leave as `None` for production, test/staging/local envs must set
-SHARE_PREPRINT_PROVIDER_PREPEND = None
```

Comment:
This also doesn't appear to be defined. Should this go into admin settings? @erinspace

Reply:
Moved to admin settings!